content/en/core/guides/docker-setup.md

```diff
@@ -35,47 +35,10 @@ CPUs: 2
 
 ## Use Docker-Compose:
 
-In the location you would like to host your configuration files, create a file titled <project_name>-medic-os-compose.yml with the following contents:
+In the location you would like to host your configuration files, create a file titled <project_name>-medic-os-compose.yml with the following contents. In this example we're creating a file called `test-docs-medic-os-compose.yml` via `curl`:
 
```

Comment:
My only thought is that not every OS you can run docker on has curl. Might add an href and a note about right clicking to save as but it's up to you really.  Also, maybe a ticket for another day. We specify a name for the file. If the assumption is the reader is first time using docker-compose they likely won't have any other compose files. 

`<a href="https://raw.githubusercontent.com/medic/cht-core/master/docker-compose.yml" >Right Click to Save the docker file. </a> `

Reply:
yeah - this is totally worth an update.  Same for the [local set up page](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/) too (deep link to raw yml).  Stay tuned

Reply:
Done! see 3cc65ec

Reply:
solid!